octoprint_mrbeam/util/uptime.py

```diff
@@ -1,14 +1,19 @@
 # coding=utf-8
-
+import os
+from sys import platform
 from datetime import timedelta
 from octoprint_mrbeam.mrb_logger import mrb_logger
 
 # http://planzero.org/blog/2012/01/26/system_uptime_in_python,_a_better_way
 def get_uptime():
     try:
-        with open("/proc/uptime", "r") as f:
-            uptime = float(f.readline().split()[0])
-        return uptime
+        if platform == 'darwin':
+            p = os.popen("uptime")
+            return p.read()
```

Comment:
Where does this come from? Is it a file??

Reply:
No, it's just a command on Mac

https://github.com/mrbeam/MrBeamPlugin/pull/1301#discussion_r681002957

Reply:
Ah ok, then `os.system()` is a more common way of handling a shell command

Reply:
I would be careful with `os.popen` because I think you have to handle a file descriptor (in this case it's `p`)

Reply:
changed